objects/app/commands.go

```diff
@@ -56,6 +56,10 @@ func Create() cli.Command {
 				Name:  "syslog-url",
 				Usage: "Syslog URL to send application logs to",
 			},
+			cli.StringFlag{
+				Name:  "architectures",
+				Usage: "The hardware architecture on which the application runs (default: x86)",
```

Comment:
Please refer the ECAR/Elaboration doc now. It has been updated.
It will good to have the documentation on par with OCI CLI to maintain consistency

Reply:
Done.